openpype/hosts/houdini/plugins/publish/validate_output_node.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.hosts.houdini.api.action import (
+    SelectInvalidAction,
+    SelectROPAction,
+)
+
+import hou
+
+
+class ValidateOutputNode(pyblish.api.InstancePlugin):
+    """Validate the instance Output Node.
+
+    This will ensure:
+        - The Output Node Path is set.
+        - The Output Node Path refers to an existing object.
```

Comment:
The docstring should explain that also a check is done that the node is either ObjNode or SopNode.